sphinx/writers/latex.py

```diff
@@ -2190,7 +2193,7 @@ def encode_uri(self, text):
 
     def visit_Text(self, node):
         text = self.encode(node.astext())
-        if not self.no_contractions:
+        if not self.no_contractions and not self.is_parsed_literal:
             text = educate_quotes_latex(text)
```

Comment:
I considered using the second argument to `educate_quotes_latex`, 

    text = educate_quotes_latex(text, ('"', '"'))

for the parsed literal case. But my feeling is that no transformation at all should be done.